exercises/armstrong-numbers/description.md

```diff
@@ -0,0 +1,9 @@
+An [Armstrong number](https://en.wikipedia.org/wiki/Narcissistic_number) is a number that is the sum of its own digits each raised to the power of the number of digits.
+
+For example:
+9 is an Armstrong number, because 9 = 9^1 = 9
+10 is *not* an Armstrong number, because 10 != 1^2 + 0^2 = 2
+153 is an Armstrong number, because: 153 = 1^3 + 5^3 + 3^3 = 1 + 125 + 27 = 153
+154 is *not* an Armstrong number, because: 154 != 1^3 + 5^3 + 4^3 = 1 + 125 + 64 = 190
```

Comment:
These line all still need some way to separate them in the rendered output, they currently run into each other in one paragraph.

see: https://github.com/pheanex/problem-specifications/blob/ff8b492d3b1863a7456c66f60fd7f646335745a9/exercises/armstrong-numbers/description.md